### PR TITLE
Add iops, throughput plugins and cluster_name support for ceph plugins

### PR DESCRIPTION
**Pull request by anmolbabu**
Add iops, cluster_name support for ceph plugins

Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
Needs to be verified

**Comment by anmolbabu** [comment]
Verified

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

File: tendrl/node_monitoring/templates/ceph_node_network_throughput.jinja [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<Plugin "python">
+    ModulePath "/usr/lib64/collectd/"
+
+    Import "ceph_node_network_throughput"
+
+    <Module "ceph_node_network_throughput">
```
Comment (r0h4n) [inline tendrl/node_monitoring/templates/ceph_node_network_throughput.jinja]: module name : "ceph_node_network_throughput"

Shouldnt these modules or plugins be prefixed with "tendrl_" ?  It makes the collectd module/plugin names more descriptive and clearly denotes that they are used by Tendrl

**Pull request merged**